acoustid/future/fingerprint.py

```
@@ -22,29 +22,43 @@
 
-def compress_fingerprint(hashes: list[int], version: int) -> bytes:
+def to_unsigned(hashes: list[int]) -> list[int]:
+    return [h & 0xFFFFFFFF for h in hashes]
+
+
+def to_signed(hashes: list[int]) -> list[int]:
+    return [h - 0x100000000 if h & 0x80000000 else h for h in hashes]
+
+
+def compress_fingerprint(
+    hashes: list[int], version: int, signed: bool = False
```

Ruff (0.8.2)

Boolean-typed positional argument in function definition

(FBT001)